android-components/components/browser/state/src/main/java/mozilla/components/browser/state/reducer/ContentStateReducer.kt

```diff
@@ -302,6 +302,12 @@ internal object ContentStateReducer {
             is ContentAction.UpdateExpandedToolbarStateAction -> updateContentState(state, action.sessionId) {
                 it.copy(showToolbarAsExpanded = action.expanded)
             }
+            is ContentAction.CheckForFormDataAction,
+            is ContentAction.ClearHighPrioritySessionAction,
+            is ContentAction.CheckForFormDataExceptionAction,
```

Comment:
How we are going handle this action? Is this going to be on the app?

Reply:
Ya Fenix will need to address the action, similar to SaveToPDF: https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/share/SaveToPDFMiddleware.kt#34

Reply:
Ok, in that case ,what are we going to show to users?  As it's not completely clear how they are going to benefit from know that the exception happened? :)

Reply:
Are they going to have a way to recover? As in the case of the `SaveToPDF` one we are notifying users because they will be waiting for the PDF to be generated, in this cases it looks like this is something that is going to happen without users know it.

Reply:
That's a good question. There wouldn't be a need to notify users that the exception happened. It'd be only helpful for the developer to know if for some unforeseen reason the GV API failed. Do you have a recommendation for how an app or Fenix should handle this exception, or maybe this exception is not necessary?

Reply:
>  Do you have a recommendation for how an app or Fenix should handle this exception, or maybe this exception is not necessary?

Maybe we could just log the info, but if we would like to keep track of how many times this happens, we could send a glean event from the app.

Reply:
Ya that sounds like a good plan. Are there examples of logging to glean in Fenix?

Reply:
Yes, we did on the Save to PDF feature https://github.com/mozilla-mobile/fenix/pull/27666/

Reply:
Sounds good, I created a separate task for this on Fenix: https://github.com/mozilla-mobile/fenix/issues/28280. Once the other comments are resolved, this patch can be merged first, without the telemetry in place, correct?

Reply:
Fenix telemetry issue and patch, once this AC patch is approved and merged: https://github.com/mozilla-mobile/fenix/pull/28444